sdk/contentunderstanding/Azure.AI.ContentUnderstanding/src/ObjectField.Extensions.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.AI.ContentUnderstanding
+{
+    /// <summary>
+    /// Extension methods and convenience properties for <see cref="ObjectField"/>.
+    /// </summary>
+    public partial class ObjectField
+    {
+        /// <summary>
+        /// Gets a field from the object by name.
+        /// </summary>
+        /// <param name="fieldName">The name of the field to retrieve.</param>
+        /// <returns>The field if found, or null if not found.</returns>
+        public ContentField this[string fieldName]
```

Comment:
Typical Dictionary behavior is to throw if key is not found.  Current behavior is arguably easier to use though without introducing TryGetValue, Contains, ...  Thoughts?